contrib/fsl_integration/sct_plugin.py

```diff
@@ -208,52 +290,703 @@ def __init__(self, parent):
+            print('Output File Name:', gmname)
+
+            #Standard output file name
+            cmd_line = "sct_deepseg_gm -i {}".format(rawimg)
+            print('Command Line:', cmd_line)
+
+        else:
+            gmname = "{}.nii.gz".format(gmimg)
+            print('Output File Name:', gmname)
+
+            #Personalized output file name
+            cmd_line = "sct_deepseg_gm -i {} -o {}".format(rawimg, gmname)
+            print('Command Line:', cmd_line)
 
         self.call_sct_command(cmd_line)
 
-        outfilename = os.path.join(os.getcwd(), 'seg.nii.gz')
+        outfilename = os.path.join(os.getcwd(), gmname)
         image = Image(outfilename)
         overlayList.append(image)
-
         opts = displayCtx.getOpts(image)
         opts.cmap = 'yellow'
 
+#Computes Cross-Sectional Area for GM, WM and GM+WM (Total)
+class TabPanelCSA(SCTPanel):
+    DESCRIPTION = """This tool computes the spinal cord cross-sectional area. 
+    For more information, please refer to the article below.<br><br>
+    <b>Specific citation</b>:<br>
+    <b>Usage</b>:<br>
+    To launch the script, upload there are two aways to do that. First, upload the raw imaging (T1w, T2w or T2s) into FSLeyes,
+    followed by the spinal cord segmentation. Always keeps this sequence, raw imaging as the first in the Overlay list field,
+    from the bottom to the top, and segmentation imaging as second. Second, just upload the segmentation imaging.
+    If you uploaded more then one image, it is not necessary uploading the images in such order, with the arrows is possible
+    to sort them and only the first imaging will be used. 
+    For more information, please refer to the article below.<br><br>
+    Martin et al.
+    <i>Can Microstructural MRI detect subclinical tissue injury in subjects with asymptomatic
+    cervical spinal cord compression? A prospective cohort study
+    (2018)</i>. BMJ Open. 13;8(4)e01980
+    """
+
+
+    def __init__(self, parent):
+        super(TabPanelCSA, self).__init__(parent=parent,
+                                          id_=wx.ID_ANY)
+        button_gm = wx.Button(self, id=wx.ID_ANY, label="Process Segmentation")
+        button_gm.Bind(wx.EVT_BUTTON, self.onButtonCSA, id = button_gm.GetId())
+
+        lbl_vertfile = ['Off', 'On']
+        self.rbox_vertfile = wx.RadioBox(self, label='Vertfile:',
+                                         choices=lbl_vertfile,
+                                         majorDimension=1,
+                                         style=wx.RA_SPECIFY_ROWS)
+
+        lbl_perlevel = ['No', 'Yes']
+        self.rbox_perlevel = wx.RadioBox(self, label='Perlevel:',
+                                         choices=lbl_perlevel,
+                                         majorDimension=1,
+                                         style=wx.RA_SPECIFY_ROWS)
+
+        hbox1 = wx.BoxSizer(wx.HORIZONTAL)
+        l1 = wx.StaticText(self, wx.ID_ANY, "Vertebral Levels/Slice Range")
```

Comment:
if this field is empty (default), i get the following error:
~~~
Command-line usage error: Option -vert needs an argument...\nAborted...\x1b[0m\n'
None
Overlay Order: [0, 2, 3, 4, 5, 6, 1]
Segmentation: /Users/julien/sct_testing_data/t2s/t2s.nii.gz
Vertebral labeling or slice range not defined.
Output Table Name: boubou
Perlevel: Yes
Error: vert box is empty.
~~~

i would suggest to use the default if this field is empty